CONTRIBUTING.md

```diff
@@ -14,26 +14,6 @@ ThisBuild / resolvers += "lightbend-akka".at("your token resolver here")
 
 ##  Build scripts
 
-1. See `.sh` scripts in the root directory.
+1. The SDK can be built with `sbt publishM2`
```

Comment:
is that enough, vs `publishLocal; publishM2`?

Reply:
publishLocal is if you want to use the artifacts from sbt (ivy), which we never use.
publishM2 should be enough for any usage from maven